package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "language-mjml",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
Please do not update the version in package.json, Atom does it automatically on publish